submission.py

```diff
@@ -57,6 +86,8 @@ def act(observation, configuration):
     _LOGGER.debug(f"State hash: '{state.state_hash}'")
     _LOGGER.debug(f"Player: '{player}'.")
     action = _catalogued_action(state, player) or _planned_action(game, state, player)
+    if _catalogued_action(state, player):
```

Comment:
this was for debugging purposes right? You probably want to remove this, or am i seeing this wrong?

Reply:
You're right, I just wanted to see if/when there were cache hits. Now that we have lru_cache, I can remove this.